src/lib/helpers/package.ts

```diff
@@ -349,7 +349,9 @@ export const getRecommendedDependencies = (packageManager: PackageManager) => {
 
   return Object.fromEntries(
     Object.entries(versionsPerPackageManager).filter(([dependencyName]) => {
-      return defaultDependencies.includes(dependencyName);
+      return defaultDependencies.some((dep) => {
+        return dep === dependencyName;
+      });
```

Comment:
Please explain this change. It looks same for me.

Reply:
I don't actually know the reason why ts started to fail here when generating docs. Previously it was fine

Reply:
https://linear.app/fluence/issue/DXJ-510/investigate-why-ts-error-happens-when-generating-docs